fusedrug_examples/requirements.txt

```diff
@@ -5,8 +5,7 @@ dgl
 pysmiles
 deepchem
 mol2vec @ git+https://github.com/samoturk/mol2vec
-
-# new depdencies for design/amp example
 modlamp
 torchtext==0.3.1 # old version of tokenizer - need to replace with another tokenizer and remove this
-biopython
\ No newline at end of file
+biopython
+x_transformers
```

Comment:
Did I use x_transformers? where?

Reply:
you import things from `fuse.dl.models.backbones.backbone_transformer` which imports it. here for example:
https://github.com/BiomedSciAI/fuse-drug/blob/c9ea278c1883bea65105cbbb26d39b07ea07e965/fusedrug_examples/design/amp/classifier/main_classifier_train.py#L42

Reply:
If I'm not missing anything,  it means that fuse is dependent on x_transformer.
This dependency is already isted here: https://github.com/BiomedSciAI/fuse-med-ml/blob/f955b484a7bec89a7a6ca27e12dfcb400353c984/fuse/requirements.txt#L32 
(to resolve you probably just needed to rerun fuse installation to get the new dependencies).